lib/rack/cors/[

```diff
@@ -0,0 +1,59 @@
+require_relative "resources/cors_misconfiguration_error"
```

Comment:
Not sure where this is coming from yet, it doesn't show in the file browser in GitHub, 404s when I click on it here and isn't on my local repo either...

Reply:
Going to dig into `.git` to work out where this is coming from..

```
git ls-files -cdm lib/rack/cors
lib/rack/cors/resource.rb
lib/rack/cors/resources.rb
lib/rack/cors/resources/cors_misconfiguration_error.rb
lib/rack/cors/result.rb
lib/rack/cors/version.rb
```